src/debug/Stream.h

```diff
@@ -88,6 +88,15 @@ class Debug
     static int override_X;
     static bool log_syslog;
 
+    /// Use the given name for debugs() messages from this helper process.
+    /// Side effect: Commits to using helper-appropriate debug levels/channels.
+    /// \sa NameThisKid()
+    static void NameThisHelper(const char *name);
```

Comment:
I do not see many of the C++ helpers being converted to use this new API. Yet all of them do debug logging (usually via the compat/debug.h API).

Please add a TODO about getting the rest converted so we do not loose track of this.

Reply:
> I do not see many of the C++ helpers being converted to use this new API.

I am not aware of any helpers other than pinger that should be converted in _this_ PR. Most helpers should not be converted until the remaining helper-related PRs are merged, and the helpers themselves start using debugs().

> Please add a TODO about getting the rest converted so we do not loose track of this.

Added `Convert all helpers to use debugs() and NameThisHelper() APIs` TODO to the PR description. I am already working on converting some of the helpers, but we need to merge the prerequisite PRs (at least) before it makes sense to post the corresponding helper conversion PRs.

Reply:
IMO we should add important TODOs to the code rather than commit message. I am not aware of anyone or tools that scan the git history for TODO comments, whereas we do often grep the code for them.

Reply:
I moved TODO from the PR description to the header file to avoid further arguments.